apps/image-server/server.js

```diff
@@ -141,6 +128,14 @@ ImageServer.on('error', (err) => {
   // Don't log 404 errors, so we do nothing here.
 });
 
+app.use(function(req, res, next) {
```

Comment:
Daarvoor heb je die const cors = require('cors'); toch al? Is dat niet dubbelop?

Reply:
De `const cors = require('cors')` regel was unused, dus die heb ik eruit gehaald!